apps/src/code-studio/pd/fit_weekend1819_registration/Welcome.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {FormGroup} from 'react-bootstrap';
+import LabeledFormComponent from '../form_components/LabeledFormComponent';
+
+export default class Welcome extends LabeledFormComponent {
+  static associatedFields = [];
```

Comment:
same question as above -  do we need this associatedFields array if it is not used in this component?